src/qml/editorwidgets/Range.qml

```diff
@@ -47,7 +47,7 @@ EditorWidgetBase {
               }
           }
 
-          inputMethodHints: Qt.ImhFormattedNumbersOnly
+          inputMethodHints: Qt.ImhFormattedNumbersOnly | Qt.ImhNoPredictiveText | Qt.ImhNoAutoUppercase | Qt.ImhSensitiveData
```

Comment:
I have tried to play with `Qt.ImhNoAutoUpperCase` before for username input on QFC login screen. No success. :(